cnd/src/db/load_swaps.rs

```diff
@@ -29,6 +29,48 @@ pub trait LoadAcceptedSwap<AL: Ledger, BL: Ledger, AA: Asset, BA: Asset> {
     ) -> anyhow::Result<AcceptedSwap<AL, BL, AA, BA>>;
 }
 
+macro_rules! impl_load_accepted_swap {
+    ($al:tt, $bl:tt, $aa:tt, $ba:tt, $accept_messages:tt, $request_messages:tt,
+    $record:tt, $select:tt) => {
+        paste::item! {
```

Comment:
We don't need this.  Its only for concatinating identifiers.  https://docs.rs/paste/0.1.6/paste/